src/components/TextField.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Styled from 'styled-components';
+
+const Text = Styled.div`
+  margin-top: ${(props) => props.mt}px;
+  margin-bottom: ${(props) => props.mb}px;
+  text-align: ${(props) => props.align}
+`;
+
+export default function TextField({
+  align,
+  children,
+  elem,
```

Comment:
Abbreviations are difficult IMO. If you have something like `char` for example, or `sm`, `md`, `lg` for sizes that might be ok. But `elem` instead of `element` doesn't save a lot of writing time. And `mt`, `mb` and `ml` are not expressive. 

When I look at this I understand what align means, but no idea about the margins. I would need to dig into the component's code to know what they are doing.

```
<TextField
  align="center"
  elem="h1"
  mt="30"
  mb="10"
>
```